### PR TITLE
Updated Rails 4.2 to 5.0 post with migrations section, regenerated site

### LINKED ISSUE #132: Rails 4.2 to 5.0: Add new section about migrations
From a comment in an article: https://www.ombulabs.com/blog/rails/upgrades/upgrade-rails-from-4-2-to-5-0.html#comment-3799159027

For this article: https://www.ombulabs.com/blog/rails/upgrades/upgrade-rails-from-4-2-to-5-0.html

We should add a section about migrations: https://www.ombulabs.com/blog/rails/upgrades/upgrade-rails-from-4-2-to-5-0.html

### PR DESCRIPTION
**Pull request by mauro-oto**
Fixes #132.

### CODE REVIEW

File: _posts/2018-03-06-upgrade-rails-from-4-2-to-5-0.markdown [hunk 1]
```diff
@@ -85,6 +85,37 @@ using `optional: true`:
 belongs_to :user, optional: true
 ```
 
+- ActiveRecord migrations now need to be tagged with the Rails version they are
+created under. One of the reasons for this is that strings in Rails 4.2 had a
+default size of 4 bytes. In Rails 5.0, their default size is now of 8 bytes.
```
Comment (etagwerker) [inline _posts/2018-03-06-upgrade-rails-from-4-2-to-5-0.markdown]: @mauro-oto Is it possible to find the PR that introduced this change? That way we could link to it here.

Comment (mauro-oto) [reply _posts/2018-03-06-upgrade-rails-from-4-2-to-5-0.markdown]: @etagwerker I wasn't able to track down the PR that introduced the change, and I checked the Rails source but I couldn't find evidence for it. I found out about this here: https://stackoverflow.com/a/35930912/2754597 which is a post on SO by Rafael Franca. Should I add a link to that or should I omit this explanation?

Comment (etagwerker) [reply _posts/2018-03-06-upgrade-rails-from-4-2-to-5-0.markdown]: @mauro-oto Please add that link. Thanks!

**Comment by mauro-oto** [comment]
@etagwerker just fixed this, thanks! :+1:

**Pull request merged**